content/docs/pipelines/troubleshooting.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "Troubleshooting"
+description = "Finding and fixing problems in your Kubeflow Pipelines deployment"
+weight = 65
++++
+
+This page presents some hints for troubleshooting specific problems that you
+may encounter.
+
+## Diagnosing problems in your Kubeflow Pipelines environment
+
+For help diagnosing environment issues that affect Kubeflow Pipelines, run
+the [`kfp diagnose_me` command-line tool](/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool).
+
+The `kfp diagnose_me` CLI reports on the configuration of your local
+development environment, Kubernetes cluster, or Google Cloud environment.
+Use this command to help resolve issues like:
+
+*  Python library dependencies
+*  Trouble accessing resources or APIs using Kubernetes secrets
+*  Trouble accessing Persistent Volume Claims
+
+To use the `kfp diagnose_me` CLI:
+
+1.  Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/).
+1.  Follow the [guide to configuring access to Kubernetes clusters][kubeconfig],
+    to update your kubeconfig with appropriate credentials and endpoint
```

Comment:
Nit: I'm not familiar with this concept (and the term is formatted like 3 different ways in 3 different contexts within the linked document), but I think maybe replacing "kubeconfig" with "kubeconfig file" could be a helpful clarification? (As opposed to, from my inference based on skimming the linked doc: the KUBECONFIG environment variable and the `kubeconfig` format)